tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +49,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="nwork", default=3, type="int",
+                      help="override number of worker threads", metavar="NUM_WORKERS")
+    parser.add_option("-t", "--timeout",
+                      dest="timeout", default=-1.0, type="float",
+                      help="override the socket connection timeout (in seconds). Negative values keep the default", metavar="TIMEOUT")
```

Comment:
Please mention that these two arguments are for the ros master.

Same below in `roslaunch/__init__.py`.